src/clients/docs_generate.zig

```diff
@@ -0,0 +1,107 @@
+const std = @import("std");
+
+const Docs = @import("./docs_types.zig").Docs;
+const go = @import("./go/docs.zig").GoDocs;
+const node = @import("./node/docs.zig").NodeDocs;
+
+const languages = [_]Docs{go, node};
+
+// pub fn run_in_docker(image: []const u8, cmds: [][]const u8) !void {
+//     var cp = std.child_process.ChildProcess.init("docker", &[_][]const u8{
+//         "run",
+//         language,
+//         },
+//                                                  );
+// }
+
+// pub fn validate_docs(language: Docs) {
+//     try run_in_docker(
+//         language.test_linux_docker_image,
+//         language.install,
+//     );
+//     try run_in_docker(
+//         language.test_linux_docker_image,
+//         language.developer_setup_bash_commands,
+//     );
+// }
+
+pub fn printHeader(writer: std.fs.File.Writer, n: i8, content: []const u8) !void {
+    var x = n;
+    while (x > 0) {
+        try writer.print("#", .{});
+        x -= 1;
+    }
```

Comment:
Can make `n` comptime and then print `"#" ** n`?

Reply:
Or maybe just write `### ` at the call-site? Would be a bit more self-explanatory

Reply:
Whoops no I see I'm wrong about that comptime order.

Reply:
Done, thanks!